bvm/ballerina-core/src/main/java/org/ballerinalang/model/values/BMap.java

```diff
@@ -282,6 +282,43 @@ public String stringValue() {
         }
     }
 
+    /**
+     * String value ignoring the access restrictions.
+     * Non-public fields of an object will also be stringified.
+     *
+     * @return string value
+     */
+    public String stringValueIgnoreAccessRestriction () {
+        readLock.lock();
+        StringJoiner sj = new StringJoiner(", ", "{", "}");
+        try {
+            switch (type.getTag()) {
+                case TypeTags.OBJECT_TYPE_TAG:
+                    for (BField field : ((BStructureType) this.type).getFields()) {
```

Comment:
Lets check on whether java 8 streams api can be used to write better code in writing this logic?

Reply:
Rewrote the logic with Java 8 streams API.